vocode/streaming/synthesizer/eleven_labs_synthesizer.py

```diff
@@ -26,6 +26,8 @@ def __init__(self, config: ElevenLabsSynthesizerConfig):
         super().__init__(config)
         self.api_key = config.api_key or getenv("ELEVEN_LABS_API_KEY")
         self.voice_id = config.voice_id or ADAM_VOICE_ID
+        self.stability = config.stability or 0.75
```

Comment:
is 0.75 the default value in their API? if `stability` / `similarity_boost` is not passed in, probably no values should be passed into the API

Reply:
yes, 0.75 is their default values. I'll change it so that no values are passed in 👍

Reply:
> is 0.75 the default value in their API? if `stability` / `similarity_boost` is not passed in, probably no values should be passed into the API

Hi, after doing some testing - it appears if `voice_settings` is passed in as a parameter, both `stability` and `similarity_boost` must also be passed in. For example - if `stability` is passed in but not `similarity_boost` the API will return an error message[0]. What would you recommend the next steps should be from here?

[0] 
![image](https://user-images.githubusercontent.com/77459359/229373837-a7004275-f3fb-49dc-989a-9657a8c35da1.png)

Reply:
what do you think of this?
- require either both parameters to be passed in or none - this can be done in `vocode.streaming.models.synthesizer.SynthesizerConfig` - see https://github.com/vocodedev/vocode-python/blob/main/vocode/streaming/models/agent.py#L37 for inspo
- if neither is passed in, then don't set the `voice_settings` param on the elevenlabs API request